api/app/schemas/chat_schemas.py

```diff
@@ -0,0 +1,40 @@
+from pydantic import BaseModel
+from uuid import UUID
+from typing import Optional
+from datetime import datetime
+
+from app.models.event import MessageRole
+from app.schemas.base_schema import BaseSchema
+
+
+class ChatCreate(BaseSchema):
+    name: str
+    description: Optional[str] = None
+    project_id: UUID
+
+
+class ChatRead(BaseSchema):
+    id: UUID
+    name: str
+    description: Optional[str] = None
+    project_id: UUID
+
+    class Config:
+        orm_mode = True
+
+
+class MessageCreate(BaseSchema):
+    role: MessageRole
```

Comment:
do we want some concept of `chat_index` - so you can put a message between existing message 3 and 4 of a chat?

Reply:
yes